src/components/split-button/split-button-colors.style.js

```diff
@@ -0,0 +1,34 @@
+export default {
```

Comment:
maybe we could separate classic colors from colors used in small/medium/large theme so it would be easier to remove classic theme?